great_expectations/experimental/datasources/filesystem_data_asset.py

```diff
@@ -31,11 +42,100 @@ class _FilesystemDataAsset(_FilePathDataAsset):
     # Filesystem specific attributes
     glob_directive: str = "**/*"
 
+    def _get_reader_options_include(self) -> set[str] | None:
```

Comment:
@NathanFarmer This module (`filesystem_data_asset.py`) is proposed to be removed in #7197 -- hence the belief that we should merge that PR first (assuming that there are no objections).  Thank you for your consideration.